app/code/core/Mage/Adminhtml/Model/System/Config/Backend/Image/Logo.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * OpenMage
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade OpenMage to newer
+ * versions in the future. If you wish to customize OpenMage for your
+ * needs please refer to https://www.openmage.org for more information.
+ *
+ * @category    Mage
+ * @package     Mage_Adminhtml
+ * @copyright  Copyright (c) RedChamps. (https://redchamps.com)
```

Comment:
this was already commented in the issue but I really think it's a bad idea to have another copyright holder in our codebase